tests/test_wps_climdex_dtr.py

```diff
@@ -12,7 +12,7 @@
         (local_path("climdexInput.rda"), "ci", "annual"),
     ],
 )
-def test_wps_climdex_mmdmt(climdex_input, ci_name, freq):
+def test_wps_climdex_dtr(climdex_input, ci_name, freq):
```

Comment:
Nice catch! This reminds me that I haven't changed the test names in other files as well from my PRs.